abdulfatai360/banka
# #164981356 Add Cashier Dashboard Template

#### What does this PR do?
Build out the template of the cashier dashboard

#### Description of Task to be completed?
- Have a section where cashier can view his/her profile
- Have a section where cashier can view details of the transactions he/she has posted

#### How should this be manually tested?
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto our local machine
- Navigate to the repo by running `cd banka/` on terminal
- Ensure your working directory is clean by running  `git status`  command on terminal
- Get a copy of the PR by typing  `git fetch origin pull/7/head:ft-add-cashier-dashboard-164981356`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-add-cashier-dashboard-164981356`
- Your directory will now be an exact copy of the PR.
- Navigate to the cashier directory by running `cd cashier/` in the terminal
- Open the `index.html` in any browser of your choice

#### What are the relevant pivotal tracker stories?
#164981356
